modules/002-deckhouse/hooks/check_deckhouse_release.go

```diff
@@ -168,6 +175,18 @@ releaseLoop:
 					notificationShiftTime = release.ApplyAfter
 				}
 			}
+			input.MetricsCollector.Expire(metricUpdatingFailedGroup)
```

Comment:
Maybe we should move expire to the start of the hook, because if we don't match switch-case `default` - metric would have improper value.

Reply:
done